src/patterns/OcAutocomplete.vue

```diff
@@ -249,7 +249,8 @@ export default {
     <div class="uk-card uk-card-default uk-card-small uk-card-body uk-margin-top">
       <oc-autocomplete v-model="complexSelection" :items="complexItems" :filter="filterComplexItems" placeholder="type 'er' for example results">
         <template v-slot:item="{item}">
-          <span>{{ item.forename }} {{ item.surname }} <em>(Age: {{ item.age }})</em></span>
+          <span class="uk-text-bold">{{ item.forename }} {{ item.surname }}</span>
+          <div class="uk-text-meta">(Age: {{ item.age }})</div>
```

Comment:
`uk-text-meta` brings it's own color, so just switch to `uk-text-small` and the problem solves itself.